image.yaml

```diff
@@ -68,10 +68,7 @@ modules:
           # Common modules from the main CE cct_module repository
           - name: dynamic-resources
             version: '1.0'
-          - name: s2i-common
-            version: '1.0'
-          - name: java-alternatives
-            version: '1.0'
+          - name: jboss.container.eap.s2i.bash
```

Comment:
@martin-kanis You will need module version here, otherwise newer Cekit will complain.